<launch>
  <!-- <arg name="file_path" default="$(find position_saver_node)/output/position.txt" /> -->
  <!-- <arg name="topic_name" default="/djiros/rtk_position" /> -->
  <!-- <arg name="topic_name" default="/fixposition/navsatfix" /> -->

  <node pkg="serial_connect" type="six_decition_radar" name="six_decition_radar" output="screen">
    <!-- <param name="file_path" value="$(find xunji)/output/position.txt" /> -->
    <!-- <param name="topic_name" value="/fixposition/navsatfix" /> -->
    <!-- <param name="topic_name" value="/djiros/rtk_position" /> -->
    <param name="targetDistance" value="230" />
    <param name="distanceThreshold" value="80.0" />
    <param name="bufferSize" value="20" />
    <param name="threshold" value="50000.0" />
    <param name="log_flag" value="1" />
  </node>

</launch>
